src/promptflow-azure/promptflow/azure/operations/_run_operations.py

```diff
@@ -142,8 +142,8 @@ def _get_run_portal_url(self, run_id: str):
         except Exception as e:
             logger.warning(f"Failed to get run portal url from pfs for run {run_id!r}: {str(e)}")
 
-        if run_info and hasattr(run_info, "studio_portal_endpoint"):
-            portal_url = run_info.studio_portal_endpoint
+        if run_info and hasattr(run_info, "studio_portal_trace_endpoint"):
```

Comment:
if a workspace/project is disabled for trace, will this url available? - we can offline sync on this

Reply:
offline synced, will add a check before print this.

Reply:
If it's disabled, this whole function won't be called.

Reply:
Updated